crates/file_store/src/store.rs

```diff
@@ -11,18 +11,26 @@ use std::{
 
 /// Persists an append-only list of changesets (`C`) to a single file.
 #[derive(Debug)]
-pub struct Store<C>
+pub struct Store<C, V = C>
 where
     C: Sync + Send,
```

Comment:
```suggestion
```

No longer needed.

Reply:
Why there were here in the first place? Change sets cannot be modified by multiple threads?

Reply:
In the past, we used a trait object that is kept internally in `Wallet` for the `PersistenceBackend`... However, I'm not sure, maybe it was just a mistake. Those bounds aren't needed for the methods we implement below.